resources/base/namespace.yaml

```
@@ -2,3 +2,10 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: minio-operator
+  labels:
+    pod-security.kubernetes.io/enforce: restricted
+    pod-security.kubernetes.io/enforce-version: latest
+    pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/audit-version: latest
+    pod-security.kubernetes.io/warn: restricted
+    pod-security.kubernetes.io/warn-version: latest
```

Could you please add a newline at the end of the file?

```suggestion
    pod-security.kubernetes.io/warn-version: latest

```